pytestqt/qt_compat.py

```diff
@@ -13,211 +13,181 @@
 from collections import namedtuple
 import os
 
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 
-if not on_rtd:  # pragma: no cover
+VersionTuple = namedtuple('VersionTuple', 'qt_api, qt_api_version, runtime, compiled')
 
-    def _try_import(name):
-        try:
-            __import__(name)
-            return True
-        except ImportError:
-            return False
+class _QtApi:
+    """
+    Interface to the underlying Qt API currently configured for pytest-qt.
 
-    def _guess_qt_api():
-        if _try_import('PyQt5'):
+    This object lazily loads all class references and other objects upon the ``set_qt_api`` method,
```

Comment:
"upon the ``set_qt_api`` method" sounds a bit strange - maybe "when the ``set_qt_api`` method is called"?